training/culture-and-process/PULL_REQUEST_GUIDANCE.md

```diff
@@ -0,0 +1,35 @@
+Pull Request Guidance
+===========================
+
+General guidance on creating a pull request
+
+### Pre reading
+
+- It is useful to have an understanding of GIT (https://git-scm.com/about) and branches as a way of working
+- Have a look at pull requests in the `dev-code-review` channel on slack
+
+### Pre requisites
+
+None
```

Comment:
I think in order to contribute you need to first setup the GPG key and configuring git to sign all commits:
https://github.com/ONSdigital/dp/blob/master/guides/GPG.md